docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -1,50 +1,44 @@
-# Prerequisite
+# `OrgUnitChangeRequest` API
 
-## Possible fields
-- `parent_id`
-- `name`
-- `opening_status`
-- `org_unit_type_id`
-- `groups`
-- `geo_point`
-- `shape`
-- `instances`
+"Change Requests" can be submitted for an `OrgUnit` and then reviewed (approved or rejected).
 
-# POST /api/orgunits/changes/?app_id=..
+This API allows the DHIS2 Health pyramid to be updated using Iaso.
 
-API used to create or modify an OrgUnit with validation (see feature flags)
-When creating an OrgUnit, the client doesn't know the ID or the OrgUnit was not created yet and we might want to apply 
-some changes still. Therefore, the IDs can be either the DB id or the UUID generated by the client.
-Changes that are not provided are considered not being part of the change request.
+The Django model that stores "Change Requests" is `OrgUnitChangeRequest`.
+
+# Create or update an `OrgUnitChangeRequest` object - Web + Mobile (IA-2421)
+
+- `POST /api/orgunits/changes/?app_id=…`
+- `PUT /api/orgunits/changes/?app_id=…`
 
 ## Permissions
 
-User must be authenticated.
+- User must be authenticated
+- User must have the `ORG_UNITS_CHANGE_REQUEST` permission
```

Comment:
This is incorrect, anyone can create a new change request but not update it

Reply:
My understanding is that we don't want change requests to be public, but limited to a set of users.

I may be wrong…

Reply:
Discussed changes are tracked here https://bluesquare.atlassian.net/browse/IA-2536.